src/main/java/com/thangle/domain/book/BookService.java

```diff
@@ -64,6 +74,21 @@ public void deleteById(final UUID id) {
         bookStore.deleteById(book.getId());
     }
 
+    public List<Book> saveDataFromExcel(final MultipartFile file) {
+        final var currentUserId = authsProvider.getCurrentUserId();
+
+        if (ImportBookHelper.hasExcelFormat(file)) {
+            try {
+                List<Book> books = ImportBookHelper.extractDataFromInputStream(file.getInputStream(), currentUserId);
+                return bookStore.saveAll(books);
+            } catch (IOException exception) {
+                throw new ForbiddenException("Fails to access excel data");
```

Comment:
Do mình không biết được thực sự có vấn đề gì đang xảy ra ở backend nên sẽ chấp nhận là lỗi 500
```suggestion
                throw new RuntimeException("Fails to access excel data", exception);
```